forumhelpers/FHP.js

```diff
@@ -1,6 +1,6 @@
 function getData(dataSet) {
 	var grabDelay = 0
-	fetch("https://theforumhelpers.github.io/forumhelpers/"+dataSet+".json")
+	fetch("../forumhelpers/"+dataSet+".json")
```

Comment:
You can't fetch from a local filesystem. This needs to point to the actual data stored on the site

Reply:
it worked locally for me